images/macos/software-report/SoftwareReport.Generator.ps1

```diff
@@ -49,10 +49,15 @@ $languageAndRuntimeList = @(
 
 if ( -not $os.IsHighSierra) {
     $languageAndRuntimeList += @(
-        (Get-GccVersion)
         (Get-FortranVersion)

```

Comment:
It looks like, we also need to add a condition for the `Get-NewmanVersion` and `Get-FortranVersion` to skip them for Monterey

Reply:
Done. Thank you.